setup.cfg

```diff
@@ -1,7 +1,6 @@
 [flake8]
 max-line-length = 110
-ignore = E133, E226, E228, E266, N802, N803, N806
-# TODO: remove E266 when Task documentation is converted to rst in DM-14207.
+ignore = E133, E226, E228, N802, N803, N806
```

Comment:
Good catch. (I wonder if CfhtIsrTask *should* have the usual docs, but adding them is clearly out of scope, so I agree with removing the unneeded exclusion).